helix-core/src/indent.rs

```diff
@@ -131,11 +135,11 @@ pub fn auto_detect_indent_style(document_text: &Rope) -> Option<IndentStyle> {
                 && prev_line_leading_count < leading_count
             {
                 if is_tabs {
-                    histogram[0] += 1;
+                    histogram[TAB_IX] += 1;
                 } else {
                     let amount = leading_count - prev_line_leading_count;
                     if amount <= MAX_INDENT as usize {
-                        histogram[amount] += 1;
+                        histogram[amount + TAB_IX] += 1;
```

Comment:
Tiny stylistic nit: I would personally prefer `TAB_IX + amount` here.  Intuitively I feel like `amount` is layered on top of `TAB_IX` to arrive at the index, rather than the other way around (even though, of course, mathematically it makes no difference).